docs/setup/independent/install-kubeadm.md

```diff
@@ -95,10 +107,12 @@ For each machine:
   deb http://apt.kubernetes.io/ kubernetes-xenial main
   EOF
   apt-get update
-  apt-get install -y kubelet kubeadm
+  apt-get install -y kubelet kubeadm kubectl
```

Comment:
wouldn't kubectl be brought in by deps ?

Reply:
Yes, but this makes it explicit to the user